### PR TITLE
feat(sentry-apps): Add Komodor to Integration Directory

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
This PR does the following:
 - Adds the weights for Komodor
 - Add Komodor's logo
 
NOTE: This requires a PR in getsentry/getsentry with a migration to add the "Deployment" label and Publish the integration.

## UI:
Screen Shot 2021-05-03 at 11 41 55 AM

### CODE REVIEW

File: static/app/types/index.tsx [hunk 1]
```diff
@@ -1497,6 +1497,7 @@ export type SentryAppComponent = {
       | 'clickup'
       | 'clubhouse'
       | 'linear'
+      | 'komodor'
```
Comment (manuzope) [inline static/app/types/index.tsx:1500]: Hmmm, I think this is a sorted list.

File: src/sentry/static/sentry/images/logos/logo-komodor.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg width="680" height="483" viewBox="0 0 680 483" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (manuzope) [inline src/sentry/static/sentry/images/logos/logo-komodor.svg:1]: Have we run this by @doralchan? She's helped me in the past making sure our SVGs are up to code.

File: src/sentry/models/integrationfeature.py [hunk 3]
```diff
@@ -13,6 +13,7 @@ class Feature:
     INCIDENT_MANAGEMENT = 5
     FEATURE_FLAG = 6
     ALERTS = 7
+    DEPLOYMENT = 8
```
Comment (manuzope) [inline src/sentry/models/integrationfeature.py:16]: I suppose we need the changes to this file so that we can add the features in the migration later?

Comment (NisanthanNanthakumar) [reply src/sentry/models/integrationfeature.py:16]: @manuzope yes, we will use it here: https://github.com/getsentry/getsentry/blob/8b202e8723fb7045178e033deddb813b455b3457/getsentry/migrations/0055_add_integration_features_komodor.py#L8-L9

File: src/sentry/static/sentry/images/logos/logo-komodor.svg (hunk 2 above)
Comment (NisanthanNanthakumar) [reply src/sentry/static/sentry/images/logos/logo-komodor.svg]: 👍  using the formatted SVG now

File: src/sentry/models/integrationfeature.py [hunk 4]
```diff
@@ -25,6 +26,7 @@ def as_choices(cls):
             (cls.INCIDENT_MANAGEMENT, "integrations-incident-management"),
             (cls.FEATURE_FLAG, "integrations-feature-flag"),
             (cls.ALERTS, "integrations-alert-rule"),
+            (cls.DEPLOYMENT, "integrations-deployment"),
```
Comment (MeredithAnya) [inline src/sentry/models/integrationfeature.py:29]: @NisanthanNanthakumar we're going to need generate a migration for this change

**Review by MeredithAnya — DISMISSED** [review]

**Review by scefali — APPROVED** [review]
The category of `Komodor` will be incorrect until we run the migration to set the feature but since the impact of that is pretty limited (we've had incorrect features for months at a time in the past), I'm OK merging as-is.

EDIT: I see it's still unpublished so no impact, we can def merge as-is.

**Pull request merged**